pbrd/pbr_vty.c

```diff
@@ -296,44 +296,6 @@ DEFPY(pbr_map_nexthop, pbr_map_nexthop_cmd,
 	return CMD_SUCCESS;
 }
 
```

Comment:
that means that by default PBRd is managing its own table identifiers.

Reply:
at the moment yes